components/tyk-bootstrap/values.yaml

```diff
@@ -27,6 +38,7 @@ global:
     dashboard: false
     # When true, sets the gateway protocol to HTTPS.
     gateway: false
+    key: EnterprisePortalLicense
```

Comment:
what does this field stand for? Can you add a comment?

Reply:
This was an useless piece of code. Deleted.